code/game/objects/items/plushes.dm

```diff
@@ -676,3 +676,23 @@
 	item_state = "moffplush"
 	attack_verb = list("fluttered", "flapped")
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+	var/souls = 0
+
+/obj/item/toy/plush/moth/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] begins staring deeply into the eyes of [src]!  It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	if(do_after(user, 50, target = user) && !user.hellbound)
+		souls++
+		switch(souls)
+			if(1 to 2)
+				desc = "A plushie depicting an unsettling mothperson. After eating [souls] [souls == 1 ? "soul" : "souls"] it's not looking so huggable now..."
+			if(3 to INFINITY)
```

Comment:
```DM
if(souls < 3)
    do baby stuff
else
    do evil souleater stuff
```

Switch just moves everything to the right for no reason but to account for 0, which is not possible at this point.